### PR TITLE
Update endpoint-protection-macos.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Do not allow user to override Gatekeeper

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3620

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by Brenduns** [comment]
#sign-off

File: memdocs/intune/protect/endpoint-protection-macos.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ Use the firewall to control connections per-application, rather than per-port. U
 
 - **Do not allow user to override Gatekeeper**
 
-  Prevents users from overriding the Gatekeeper setting, and prevents users from Control clicking to install an app. When enabled, users can Control-click any app, and install it.
+  Prevents users from overriding the Gatekeeper setting, and prevents users from Control clicking to install an app. When enabled, users cannot Control-click any app, or install it.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/endpoint-protection-macos.md:148]:
```suggestion
  Prevents users from overriding the Gatekeeper setting, and prevents users from Control-clicking to install an app. When enabled, users cannot Control-click any app, or install it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: memdocs/intune/protect/endpoint-protection-macos.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ Use the firewall to control connections per-application, rather than per-port. U
 
 - **Do not allow user to override Gatekeeper**
 
-  Prevents users from overriding the Gatekeeper setting, and prevents users from Control clicking to install an app. When enabled, users can Control-click any app, and install it.
+  Prevents users from overriding the Gatekeeper setting, and prevents users from Control-clicking to install an app. When enabled, users cannot Control-click any app, or install it.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/endpoint-protection-macos.md:148]:
```suggestion
  Prevents users from overriding the Gatekeeper setting, and prevents users from Control-clicking to install an app. When enabled, users cannot Control-click any app to install it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**